pkg/webui/console/views/application-general-settings/connect.js

```diff
@@ -162,10 +162,7 @@ const addHocs = pipe(
   withFeatureRequirement(mayEditBasicApplicationInfo, {
     redirect: ({ appId }) => `/applications/${appId}`,
   }),
-  withRequest(
-    ({ loadData }) => loadData(),
-    ({ fetching }) => fetching,
-  ),
+  withRequest(({ loadData }) => loadData()),
```

Comment:
`error` is not selected here as well, even though it gets composed in L86.

Reply:
Would it be enough to add `error` to the returned values here https://github.com/TheThingsNetwork/lorawan-stack/blob/b1e0a11a620ba04a1e8bf4edd7d6c3cca2904147/pkg/webui/console/views/application-general-settings/connect.js#L109?

Reply:
Yes exactly.